main.go

```diff
@@ -53,6 +53,7 @@ import (
 	placementv1 "github.com/openstack-k8s-operators/placement-operator/api/v1beta1"
 	swiftv1 "github.com/openstack-k8s-operators/swift-operator/api/v1beta1"
 	telemetryv1 "github.com/openstack-k8s-operators/telemetry-operator/api/v1beta1"
+	_ "github.com/openstack-k8s-operators/test-operator/api/v1beta1"
```

Comment:
while the dash here implies meaning, it might be worth adding an explicit comment on this one so that someone doesn't come along and remove it afterwards

Reply:
Got it, thank you, done!